### PR TITLE
Political donation

### PR DESCRIPTION
**Pull request by sboisvert**
This doesn't use the new mixin yet. I want to get it in to stop the merge conflicts ;).
The big difference here with the other pages is that we have 2 amount fields on one page. 
To handle this we have a custom sanitizer that allows empty values (counts them as 0).
Currently this supports empty values for none, either, or both fields. This is in contrast with other amount pages where 0 is not a valid amount. 
I think this is fine for now but user testing might let us know if we need to change it to be like the others and force one valid value OR if we should change the others to accept 0 as a valid value.

### CODE REVIEW

File: views/deductions/political.pug [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+block content
+
+  h1 #{title}
+
+  div
+    p #{__('Did you make any political contributions during the current tax year (filing for 2018)?')}
+
+  form.cra-form(method='post')
+    +yesNoRadios('politicalClaim', politicalClaim, 'Deduct political contributions?', errors)
```
Comment (pcraig3) [inline views/deductions/political.pug]: Great! Easy.

File: views/deductions/political.pug [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    input#redirect(name='redirect', type='hidden', value='/deductions/donations')
+
+    .buttons
```
Comment (pcraig3) [inline views/deductions/political.pug]: Again, we can use the button mixin here too.

File: views/personal/maritalStatus.pug [hunk 3]
```diff
@@ -20,6 +20,6 @@ block content
           td
             div #{data.personal.maritalStatus}
 
-  +linkButtons('/deductions/donations')
+  +linkButtons('/deductions/political')
```
Comment (pcraig3) [inline views/personal/maritalStatus.pug]: 👍

File: views/deductions/political.pug (hunk 2 above)
Comment (sboisvert) [reply views/deductions/political.pug]: done

File: views/deductions/political-amount.pug [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    div(class={['has-error']: errors && errors.politicalProvincialAmount})
+      label#politicalProvincial__label(for='politicalProvincialAmount') #{__('Total provincial contributions')}
+      if errors && errors.politicalProvincialAmount
+        +validationMessage(errors.politicalProvincialAmount.msg, 'politicalProvincial')
+      span#political__unit.input-with-unit__unit(for='politicalProvincialAmount') $
+      input.w-1-2.input-with-unit#political(name='politicalProvincialAmount', type='text', placeholder="0.00", autocomplete='off' aria-describedby=(errors && errors.politicalProvincialAmount ? 'politicalProvincialAmount-error' : false) aria-labelledby="politicalProvincialAmount__label political__unit")
+
+    input#redirect(name='redirect', type='hidden', value='/deductions/donations')
+
+    .buttons
```
[the remark this answers is not in the document]
Comment (sboisvert) [reply views/deductions/political-amount.pug]: boom

File: views/deductions/political-amount.pug [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        +validationMessage(errors.politicalFederalAmount.msg, 'politicalFederal')
+      span#political__unit.input-with-unit__unit(for='politicalFederalAmount') $
+      input.w-1-2.input-with-unit#political(name='politicalFederalAmount', type='text', placeholder="0.00", autocomplete='off' aria-describedby=(errors && errors.politicalFederal ? 'politicalFederalAmount-error' : false) aria-labelledby="politicalFederalAmount__label political__unit")
+    div(class={['has-error']: errors && errors.politicalProvincialAmount})
+      label#politicalProvincial__label(for='politicalProvincialAmount') #{__('Total provincial contributions')}
+      if errors && errors.politicalProvincialAmount
+        +validationMessage(errors.politicalProvincialAmount.msg, 'politicalProvincial')
+      span#political__unit.input-with-unit__unit(for='politicalProvincialAmount') $
+      input.w-1-2.input-with-unit#political(name='politicalProvincialAmount', type='text', placeholder="0.00", autocomplete='off' aria-describedby=(errors && errors.politicalProvincialAmount ? 'politicalProvincialAmount-error' : false) aria-labelledby="politicalProvincialAmount__label political__unit")
```
[the remark this answers is not in the document]
Comment (sboisvert) [reply views/deductions/political-amount.pug]: done

File: views/deductions/political-amount.pug [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+  h1 #{title}
+
+  div
+    p #{__('Please enter your political contributions into the fields below.')}
+
+  form.cra-form(method='post')
+    +textInput('Total federal contributions')(class='w-1-2 input-with-unit' name='politicalFederalAmount' placeholder='0.00' aria-labelledby='politicalFederalAmount__unit politicalFederalAmount__label' value=politicalFederalAmount)
+ 
+    +textInput('Total provincial contributions')(class='w-1-2 input-with-unit' name='politicalProvincialAmount' placeholder='0.00' aria-labelledby='politicalProvincialAmount__unit politicalProvincialAmount__label' value=politicalProvincialAmount)
```
Comment (pcraig3) [inline views/deductions/political-amount.pug]: Nice, still some copy+paste but overall way easier to look at  💯

**Pull request merged**